src/lab/CustomText/base.ts

```diff
@@ -0,0 +1,14 @@
+export interface CustomTextProps {
+  value: string;
+  width: string;
+  onChangeText: (value: string) => void;
+  initialStateOutline?: boolean;
+  showLabelOnDisabled?: boolean;
+  showHelperTextOnDisabled?: boolean;
+  validateText?: boolean;
+  placeholder?: string;
+  label?: string;
+  helperText?: string;
+  multiline?: boolean;
+  isEditable?: boolean;
```

Comment:
A lot of these can be used from MUI's Props, thus this can be optimized

Reply:
the component has been updated. InputField component has been utilized to optimize